### ISSUE TITLE
Indent scribble like DrRacket.

### ISSUE THREAD

**Issue by LeifAndersen** [greghendershott/racket-mode#206]
DrRacket indents scribble files so that they are easy to read. (Given that you likely want prose in at at-exp anyway.)

For example, I can create a file, and it indents like this:

```
#lang scribble/base

@list{This is
 a long bunch of words}
```

But in racket-mode, the following indents like this:

```
#lang scribble/base

@list{foo
      bar}
```

Which, leads to text going way to far off to the left.

**Comment by greghendershott** [comment]
See also: #64

**Comment by greghendershott** [comment]
Recently when I was converting Frog's README into Scribble, which was a pretty big job (for me), I found [`scribble-mode`](https://melpa.org/#/scribble-mode) to work well for extended prose editing.

Sometimes I would even `M-x racket-mode` and `M-x scribble-mode` to flip back and forth. (But mainly, I triggered `make` to rebuild the docs, as opposed to `racket-run`-ing the file, and stayed in `scribble-mode`.)

Also I think the status quo indentation for at-exprs is quite nice for e.g. `#lang at-exp racket` (as opposed to authoring Scribble). 

So ... I think the moral of the story might be, it's great that we can use one or more Emacs major modes to edit a given file in the way(s) we want?  And, it's unlikely that one major mode will do the desired thing for all files and all people -- so, it's OK to have each of them prioritize?
